### PR TITLE
rules_fuzzing@0.5.1

### LINKED ISSUE #1728: wanted: bazelbuild/rules_fuzzing
### Module location

https://github.com/bazelbuild/rules_fuzzing

### Link to bzlmod issue in the module's repository

https://github.com/bazelbuild/rules_fuzzing/issues/189
https://github.com/bazelbuild/rules_fuzzing/pull/242

### Any other context to provide?

_No response_

### Fund our work

- [ ] Sponsor our community's open source work by donating a [feature bounty](https://opencollective.com/bazel-rules-authors-sig/projects/bazel-central-registry)

### PR DESCRIPTION
**Pull request by mmorel-35**
Release: https://github.com/bazelbuild/rules_fuzzing/releases/tag/v0.5.1

Closes #1728

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (rules_fuzzing) have been updated in this PR. Please review the changes.

**Review by fmeum — COMMENTED** [review]
Thanks for the contribution! @stefanbucur is currently working on the last steps for setting up the automated BCR publishing flow for rules_fuzzing. Let's wait for a bit before we add this manually.

**Comment by fmeum** [comment]
> If it requires a new version, this one can be added or am I missing something ?

That's true, but we need to ensure that the next release is API compatible. That's why I would at least want to wait for @stefanbucur's approval.

**Comment by fmeum** [comment]
@meteorcloudy Would he be okay with me becoming the BCR maintainer?

**Comment by stefanbucur** [comment]
> Yes, that's definitely fine!

+1, I think Fabian would be most qualified to maintain `rules_fuzzing` moving forward. Nowadays I have very little time on my end and I often am the bottleneck in reviewing & approving PRs too.

**Comment by fmeum** [comment]
@mmorel-35 Could you add my contact info (you can get it from rules_go) here instead of Stefan's? You also have to exclude the `replay` test as it pulls in googletest. Then we can merge this, thanks for working on it!

File: modules/rules_fuzzing/0.5.1/presubmit.yml
Comment (fmeum) [inline modules/rules_fuzzing/0.5.1/presubmit.yml:11]: @mmorel-35 You can specify target patterns with a leading `-` here to exclude them.

**Comment by keith** [comment]
would be great if we could merge this so i could use it in https://github.com/bazelbuild/bazel-central-registry/pull/1807

**Comment by stefanbucur** [comment]
> would be great if we could merge this so i could use it in #1807

@fmeum Is there something on my end that you folks are blocked on?

**Comment by fmeum** [comment]
@stefanbucur No, I just didn't realize that this was updated and the checks were failing. 

Bazel 6 support is lacking due to a bug fixed by https://github.com/bazelbuild/rules_fuzzing/pull/248, but this can wait for a new release. Testing the main repo itself was buggy for various reasons, but that's to be expected. Just testing the test module makes more sense and that works.

**Review by fmeum — APPROVED** [review]

**Comment by keith** [comment]
thanks!

**Pull request merged**
